domains/Auth/SignIn/index.js

```diff
@@ -112,9 +113,35 @@ const SignIn = ({ navigation }) => {
                       // credentials saved do not match those entered, overwrite saved credentials
                       if (userCreds === null || values.username !== userCreds.username
                         || values.password !== userCreds.password) {
-                        handleSaveCredentials(values);
+                        // Store user organization
```

Comment:
This is the function i attempted to modularize in domains/auth/storeOrganization issue is documented in file. probably an easy fix.